src/wallet/fees.cpp

```diff
@@ -76,6 +76,62 @@ CAmount GetMinimumFee(unsigned int nTxBytes, const CCoinControl& coin_control, c
     return fee_needed;
 }
 
+CFeeRate GetRequiredFeeRate()
+{
+    return std::max(CWallet::minTxFee, ::minRelayTxFee);
+}
+
+CFeeRate GetMinimumFeeRate(const CCoinControl& coin_control, const CTxMemPool& pool, const CBlockPolicyEstimator& estimator, FeeCalculation *feeCalc)
```

Comment:
Should just implement original `GetMinimumFee` & `GetRequiredFee` functions in terms of these new feerate functions. Not good to duplicate so much code.

Reply:
`GetMinimumFee` has `maxTxFee` logic that this doesn't, so the wrapper would have to include that.